src/token/api/src/error.rs

```diff
@@ -32,14 +26,13 @@ pub enum TxError {
     AmountOverflow,
     #[error("Account is not found")]
     AccountNotFound,
-    #[error("{message}")]
-    GenericError { message: String },
-    #[error("Claim not Allowed")]
-    ClaimNotAllowed,
-    #[error("Temporary unavailable")]
-    TemporaryUnavailable,
+    #[error("No claimable tokens are on the requested subaccount")]
```

Comment:
```suggestion
    #[error("no tokens to claim on the requested subaccount.")]
```
Also consistency: all error messages should be lower cased